csrankings.csv

```diff
@@ -9953,6 +9953,7 @@ Mohamed Y. Eltabakh,Worcester Polytechnic Institute,http://www.wpi.edu/academics
 Mohamed Yehia Dahab,King Abdulaziz University,http://www.kau.edu.sa/Contacts.aspx?Site_ID=611&Lng=EN&Cont=18607,YExgqdMAAAAJ
 Mohamed Younis,Univ. of Maryland - Baltimore County,http://www.csee.umbc.edu/~younis,Q4MvxMIAAAAJ
 Mohamed Zahran,New York University,http://www.mzahran.com,Z2tBw84AAAAJ
+Mohammad A. Islam 001,University of Texas at Arlington,http://crystal.uta.edu/~mislam/,rxKWd1cAAAAJ

```

Comment:
Should be "0001" and needs an alias entry as well.
https://dblp.uni-trier.de/pers/hd/i/Islam_0001:Mohammad_A=

Reply:
Updated accordingly in csrankings.csv and dblp-aliases.csv